go/example.go

```diff
@@ -188,13 +192,72 @@ func GetOpenOrders(jwtToken string) []*Order {
 
 	// Response
 	res, _ := http.DefaultClient.Do(req)
-	Print(res.Status)
+	Print("GET /orders:", res.Status)
 
 	orders := ParseGetOrders(res)
 	return orders
 }
 
-func ExampleSignOrder() {
+func SubmitOrder(
+	config SystemConfigResponse,
+	dexAccountAddress string,
+	dexPrivateKey string,
+	jwtToken string,
+) {
+	dexAccountAddressBN := types.HexToBN(dexAccountAddress)
+
+	sc := caigo.StarkCurve{}
+	typedData, _ := NewVerificationTypedData("Order", config.ChainId)
+	domEnc, _ := typedData.GetTypedMessageHash("StarkNetDomain", typedData.Domain, sc)
+
+	// Change order values here
+	market := "ETH-USD-PERP"
+	side := "BUY"
+	orderType := "LIMIT"
+	size := "1"
+	price := "900"
+	timestamp := time.Now().UnixMilli()
+
+	orderPayload := &OrderPayload{
+		Timestamp: timestamp,
+		Market:    market,
+		Side:      side,
+		OrderType: orderType,
+		Size:      size,
+		Price:     price,
+	}
+	messageHash, _ := GnarkGetMessageHash(typedData, domEnc, dexAccountAddressBN, orderPayload, sc)
+	r, s, _ := GnarkSign(messageHash, dexPrivateKey)
+
+	// URL
+	ordersUrl := fmt.Sprintf("%s/orders", PARADEX_HTTP_URL)
+
+	// Body
+	body := OrderRequest{
+		Market:             market,
+		Side:               OrderSide(side),
+		Type:               OrderType(orderType),
+		Size:               size,
+		Price:              price,
+		Signature:          GetSignatureStr(r, s),
+		SignatureTimestamp: timestamp,
+	}
+	bodyByte, _ := json.Marshal(body)
```

Comment:
may worth to check the error, just in case the user introduces some wrong value; would be easier to understand the issue